test/dd/test_package.cpp

```diff
@@ -1579,7 +1590,7 @@ TEST(DDPackageTest, DDFromSingleQubitMatrix) {
   const auto dd = std::make_unique<dd::Package<>>(nrQubits);
   const auto matDD = dd->makeDDFromMatrix(inputMatrix);
 
-  const auto outputMatrix = matDD.getMatrix();
+  const auto outputMatrix = dd->getMatrix(matDD, nrQubits);
```

Comment:
Similar to above, this is using a function that should no longer exist. Please also check all other occurrences within this file.